src/aomusicplayer.cpp

```diff
@@ -125,7 +132,27 @@ int AOMusicPlayer::play(QString p_song, int channel, bool loop,
 
   this->set_looping(loop, channel); // Have to do this here due to any
                                     // crossfading-related changes, etc.
-  return BASS_ErrorGetCode();
+
+  bool is_stop = (p_song == "~stop.mp3");
+  QString p_song_clear = p_song.left(p_song.lastIndexOf("."));
+  p_song_clear = p_song_clear.right(p_song_clear.length() - (p_song_clear.lastIndexOf("/") + 1));
+  int error_code = BASS_ErrorGetCode();
+
+  if (is_stop) {
+    return "None";
```

Comment:
```suggestion
    return QObject::tr("None");
```
reason this needs to use `QObject::tr` instead of `tr` is that this object isn't marked as a Q_OBJECT